pom.xml

```diff
@@ -132,6 +132,18 @@
     </plugins>
   </build>
 
+  <dependencyManagement>

```

Comment:
This is already defined in the parent POM. Is it not working for you?

Reply:
i was rebasing 2-3 year old branches and had not spotted it move into the commons-parent. will correct all the branches i've made.